### PR TITLE
Temporarily disable join button to prevent navigation to URL

### LINKED ISSUE #570: Disable join button
Disable join button, change color to grey and put the text "We're currently not accepting new applications. Please check back on October 1 for availability".

### PR DESCRIPTION
**Pull request by ishant005**
### Issue:

Closes #570 

### What is the change?

Disabled the join button .

- Disabled join button
- Added a new line below the join button

### Is Development Tested?

- [x] Yes
- [ ] No

### Before / After Change Screenshots

Before :
Screenshot (33)

After :
Screenshot (36)

Demo Video

https://github.com/Real-Dev-Squad/website-www/assets/91171097/49b5d2a3-55f3-4514-9bb1-e59b8033649f

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -200,6 +201,12 @@ body {
   justify-content: center;
   align-items: center;
   margin-top: 20px;
+  background: var(--color-button-disable);
+  pointer-events: none;
```
Comment (shubhamsigdar1) [inline css/main.css:205]: `cursor: not-allowed;`

**Review by shubhamsigdar1 — COMMENTED** [review]
Great work @ishant005

**Comment by shubhamsigdar1** [comment]
I seen in video when you clicking on button something is loading??

File: js/index.js [hunk 2]
```diff
@@ -21,6 +21,10 @@ for (let i = 0; i < numOfMembers; i++) {
   memberLinks.appendChild(memberTags);
   memberSection.appendChild(memberLinks);
 }
+const joinButtonDisable = document.getElementById('button-disable');
+joinButtonDisable.addEventListener('click', (event) => {
+  event.preventDefault();
+});
```
Comment (shubhamsigdar1) [inline js/index.js:27]: Hey Why you are doing this all??
https://www.w3schools.com/tags/att_button_disabled.asp
Please check above link

Using the disabled attribute on a <button> tag in HTML has several benefits, mainly related to user experience and accessibility. When you add the disabled attribute to a button element, it prevents users from interacting with or clicking the button.

**Review by shubhamsigdar1 — COMMENTED** [review]
Please can you show me responsiveness also in video

File: js/index.js (hunk 2 above)
Comment (ishant005) [reply js/index.js:27]: Disabled is working for button .But it is not working anchor tag. So when we apply disabled on button ,it is still clickable .
That's why we need to add JS code

File: index.html [hunk 3]
```diff
@@ -261,9 +261,15 @@ <h1 class="features-title">How can you join?</h1>
             you don’t know much yet, but it won’t be okay to notput in any
             efforts for yourself. We want to value everyone’s time and efforts.
           </p>
-          <a class="btn-join" href="https://realdevsquad.com/join">
-            Join the Squad
-          </a>
+
+          <button class="btn-join" id="button-disable">
+            <a href="https://realdevsquad.com/join"> Join the Squad </a>
+          </button>
```
Comment (sahsisunny) [inline index.html:267]: Can you please explain why the anchor was changed to a button?

File: js/index.js (hunk 2 above)
Comment (sahsisunny) [inline js/index.js:27]:
```suggestion
const joinButtonDisable = document.getElementById('button-disable');
joinButtonDisable.disabled = true;
```

File: index.html (hunk 3 above)
Comment (ishant005) [reply index.html]: removed button and using only anchor tag

File: js/index.js (hunk 2 above)
Comment (ishant005) [reply js/index.js]: When we use pointer-events:none ,there is no use of JS code,I guess.

**Review by rohan09-raj — APPROVED** [review]

**Review by sahsisunny — APPROVED** [review]

**Pull request merged**